megamek/src/megamek/common/BattleArmor.java

```diff
@@ -589,6 +589,16 @@ public String getMovementAbbr(EntityMovementType mtype) {
     @Override
     public HitData rollHitLocation(int table, int side, int aimedLocation, AimingMode aimingMode,
                                    int cover) {
+        return rollHitLocation(table, side, aimedLocation, aimingMode, cover, false);
+    }
+
+    /**
+     * Battle Armor units can only get hit in undestroyed troopers.
+     *
+     * @param isAttackingConvInfantry Set to true when attacked by CI, as these cannot score TacOps crits
+     */
+    public HitData rollHitLocation(int table, int side, int aimedLocation, AimingMode aimingMode,
+                                   int cover, boolean isAttackingConvInfantry) {
```

Comment:
Why add this method with two unused parameters? Keeping it consistent with the general rollHitLocation methods?

Reply:
More like, didn't think about it. I guess it's not worth getting the compiler warnings and github alerts for it, so I removed them.